File: ProgTeam Spring Week 1/templates/B/B_acm.java
import java.io.*;
import java.util.*;

class Main{
    // Make sure your class name is Main when you submit if using this template!

    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;
        int time_solved;
        int score = 0, penalty = 0;
        // Add any auxiliary variables you need here
        while(true){
            st = new StringTokenizer(br.readLine());
            time_solved = Integer.parseInt(st.nextToken());
            if(time_solved == -1) break;
            char problem = st.nextToken().charAt(0);
            String correct = st.nextToken();
            if(correct.equals("right")){
                // Your code here
            }else{
                // Your code here
            }
        }
        System.out.println(score + " " + penalty);
    }
}

File: ProgTeam Spring Week 1/templates/B/B_acm.py
# Place any auxilary variables you need here
time_solved = 0
score, penalty = 0, 0
while True:
    line = input()
    if line == "-1":
        break
    time_solved, problem, correct = line.split()
    time_solved = int(time_solved)
    if correct == "right":
        # Your code here
    else:
        # Your code here
    
print(score, penalty)